### ISSUE TITLE
FRESH schema documentation

### ISSUE THREAD

**Issue by kcallis** [hacksalot/HackMyResume#232]
I wanted to get a better understanding of the particular of the schema, so I attempted to browse http://resume.freshstandards.org and found that it was dead. I tried a couple of variations and still no joy. Is there another location that I should be looking or should I just assume that that seemed like an ideal, but I should go back to json resume format?

**Comment by 501st-alpha1** [comment]
Yeah, as was [mentioned in #229](https://github.com/hacksalot/HackMyResume/issues/229#issuecomment-598825487), the websites related to this project have been down for a few years now.  However, the FRESH resume schema was also [published on GitHub](https://github.com/fresh-standard/fresh-resume-schema).
